doc/whats_new/v1.1.rst

```diff
@@ -451,6 +451,11 @@ Changelog
   parameters in `fit` instead of `__init__`.
   :pr:`21436` by :user:`Haidar Almubarak <Haidar13 >`.
 
+- |Enhancement| :func:`svm.SVR`, :func:`svm.SVC` and other support vector
+  based classes which use dual gap estimation shows error
```

Comment:
We can list out all the estimators that is a subclass of `BaseLibSVM` so users know exactly which estimators changed.